src/index.html

```diff
@@ -1,22 +1,126 @@
 <!doctype html>
 <html lang="en">
   <head>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
```

Comment:
There are duplicate `<link>` tags for preconnecting to the same resources (`https://fonts.googleapis.com` and `https://fonts.gstatic.com`). It's unnecessary to have multiple preconnect links for the same resource. Consider removing the redundant tags to optimize the HTML.